service/test/controllers/health/HealthControllerTest.java

```diff
@@ -15,58 +15,49 @@
 import play.mvc.Result;
 import play.test.Helpers;
 
-@Ignore
 public class HealthControllerTest extends BaseControllerTest {
-  TestHelper testHelper;
   public static Application app;
-  public static Map<String, String[]> headerMap;
 
   @Before
-  public void setUp() throws Exception {
-
-    testHelper = new TestHelper();
+  public void setUp() {
     app = Helpers.fakeApplication();
-    Helpers.start(app);
-    headerMap = testHelper.getHeaderMap();
   }
 
   @After
-  public void tearDown() throws Exception {
-    headerMap = null;
+  public void tearDown() {
     app = null;
-    testHelper = null;
   }
 
-  @Ignore
+
   @Test
   public void testGetHealthSuccess() {
     Map<String, Object> reqMap = new HashMap<>();
     reqMap.put("accept", "yes");
-    Result result = testHelper.performTest("/health", "GET", reqMap, headerMap);
+    Result result = testHelper.performTest("/health", "GET", reqMap, headerMap, app);
     assertTrue(testHelper.getResponseStatus(result) == Response.Status.OK.getStatusCode());
   }
 
   @Test
   public void testGetHealthFailure() {
     Map<String, Object> reqMap = new HashMap<>();
     reqMap.put("accept", "yes");
-    Result result = testHelper.performTest("/health", "POST", reqMap, headerMap);
+    Result result = testHelper.performTest("/health", "POST", reqMap, headerMap, app);
     assertTrue(testHelper.getResponseStatus(result) == Response.Status.NOT_FOUND.getStatusCode());
   }
 
   @Test
   public void testGetUserOrgServiceHealthSuccess() {
```

Comment:
testGetUserOrgServiceHealthSuccess will be testCompleteHealthSuccess

Reply:
Done.